### DISCUSSION TITLE
add peer

### DISCUSSION THREAD

**Discussion by jadsongmatos** [subins2000/p2pt#45] [Q&A]
can I add peer knowing only its id?

**Comment by subins2000** [comment]
Can you elaborate your question a bit more ?

**Comment by jadsongmatos** [comment]
I was testing p2chat and had the idea to create a channel called tracker, this channel would be used by bots.
Bots would constantly pick up addresses from peers passing through the tracker channel.
When the user is looking for the pairs of some channel he could send a message on that channel, if any bot has this information it would respond with a list of pairs.
when I say address, I mean id or something that can be used to create a connection.
the idea to work together with the current tracker system taking some of the weight off.
